src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -387,18 +387,13 @@ private async Task<IEnumerable<JObject>> GetFunctionTriggers(IEnumerable<Functio
         internal static HttpRequestMessage BuildSetTriggersRequest()
         {
             var protocol = "https";
-            // On private stamps with no ssl certificate use http instead.
             if (Environment.GetEnvironmentVariable(EnvironmentSettingNames.SkipSslValidation) == "1")
             {
+                // On private stamps with no ssl certificate use http instead.
                 protocol = "http";
             }
 
-            var hostname = Environment.GetEnvironmentVariable(EnvironmentSettingNames.AzureWebsiteHostName);
-            // Linux Dedicated on AppService doesn't have WEBSITE_HOSTNAME
```

Comment:
This Linux Dedicated fallback I kept in the code. On Windows, HostName should never be missing, but if it is, we'll have a decent fallback